style.css

```diff
@@ -120,12 +125,221 @@
 .hamburger {
   text-decoration: none;
   list-style: absolute;
-  position: fixed;
-  padding-left: 300px;
-  left: 16.67%;
-  right: 16.67%;
+  position: absolute;
+  left: 68%;
   font-size: larger;
+  font-weight: bold;
   border: none;
   color: #6070ff;
   background-color: transparent;
 }
+
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  gap: 50px;
+}
+
+.works-1 > div {
+  border: 1px solid black;
+}
+
+.work-card {
+  background-color: #fff;
+  border-radius: 24px;
+  border: 1px solid #dfe1e6;
+  padding-top: 10px;
+  margin-top: 20px;
+  margin-bottom: 42px;
+  width: 360px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 13px;
+}
+
+.work-card-img {
+  object-fit: cover;
+  align-items: center;
+  justify-content: center;
+  margin-top: 12px;
+}
+
+.align-center-text { /* Tonic */
+  padding: 30px 25px;
+  margin-top: -40px;
+}
+
+.align-center-text-1 { /* Multi-Post Stories */
+  padding: 30px 25px;
+  margin-top: px;
+}
+
+.align-center-text-2 {/* About Myself */
+  padding: 30px 5px;
+}
+
+.about-section-2 {
+  padding: 5px 30px;
+  width: 310px;
+  width: 300px;
+  margin: 0;
+  margin-left: 13px;
+}
+
+.badges {
+  align-items: flex-start;
+  justify-content: start;
+  margin-left: -39px;
+}
+
+.works-badge {
+  text-align: start;
+  padding: 0.4rem 1.2rem;
+  background: #ebebff;
+  border-radius: 0.8rem;
+  list-style: none;
+  display: inline;
+  color: var(--color-primary);
+  font-size: 12px;
+  font-family: "poppins", sans-serif !important;
+}
+
+.work-card p {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.canopy {
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #172b4d;
+}
+
+.dev-date {
+  color: #6b778c;
+  font-weight: 600 !important;
+}
+
+.bullet {
+  padding-left: 15px;
+  color: #c1c7d0;
+  font-size: 20px;
+}
+
+.works-content-p {
+  color: var(--color-text);
+}
+
+.work-heading {
+  height: 20px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 33px;
+  color: #091e42;
+}
+
+.work-heading-2 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 33px;
+  color: #091e42;
+  padding: 0;
+  width: 190px;
+  margin: 0;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  font-family: "Poppins", sans-serif !important;
+  font-style: normal;
+  font-weight: 500;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 7px;
+  border-color: var(--color-primary);
+  background-color: transparent;
+}
+
```

Comment:
- [ ] Kindly implement all button interactions (enable, disabled, active) as it is requested in project requirements. It will increase user experience as well :wink:

   - ### Figma design
   
   ![image](https://user-images.githubusercontent.com/87197412/185871944-d0690fa4-c18e-4682-9da2-9121d8da96d1.png)